packages/select/src/FilterMultiSelect/components/ListBox/ListBox.tsx

```diff
@@ -65,7 +65,7 @@ export const ListBox = ({ children }: ListBoxProps): JSX.Element => {
   if (hasNoItems) {
     return (
       <>
-        <div>{children(itemsState)}</div>
+        <div>{!itemsState && children(itemsState)}</div>
```

Comment:
Not entirely sure why, but typescript started throwing errors for various parts of `FilterMultieSelect`